server/runtime/src/main/java/org/infinispan/server/Extensions.java

```diff
@@ -4,33 +4,43 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import javax.script.ScriptEngineFactory;
+
 import org.infinispan.commons.util.ServiceFinder;
 import org.infinispan.filter.KeyValueFilterConverterFactory;
 import org.infinispan.filter.NamedFactory;
+import org.infinispan.manager.EmbeddedCacheManager;
 import org.infinispan.notifications.cachelistener.filter.CacheEventConverterFactory;
 import org.infinispan.notifications.cachelistener.filter.CacheEventFilterConverterFactory;
 import org.infinispan.notifications.cachelistener.filter.CacheEventFilterFactory;
 import org.infinispan.server.hotrod.HotRodServer;
+import org.infinispan.server.tasks.ServerTaskEngine;
+import org.infinispan.server.tasks.ServerTaskWrapper;
+import org.infinispan.tasks.ServerTask;
+import org.infinispan.tasks.spi.TaskEngine;
 
 /**
  * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
  * @since 10.0
  **/
 public class Extensions {
-   private final Map<String, CacheEventFilterFactory> filterFactories = new HashMap<>();
-   private final Map<String, CacheEventConverterFactory> converterFactories = new HashMap<>();
-   private final Map<String, CacheEventFilterConverterFactory> filterConverterFactories = new HashMap<>();
-   private final Map<String, KeyValueFilterConverterFactory> keyValueFilterConverterFactories = new HashMap<>();
+   private final Map<String, CacheEventFilterFactory> filterFactories = new HashMap<>(2);
```

Comment:
I would say just leave them how they were before. Unfortunately this will cause an additional resize due to a load factor of .75.

Reply:
Done